nazaroff добавлено дз за 01-11

nazaroviktor2 opened the pull request:
@siriusdevs

nazaroviktor2 committed:
добавлено дз за 01-11

siriusdevs commented on 01-11/models/__init__.py:
```diff
@@ -0,0 +1,4 @@
+"""Init file."""
```
Это что за файл с комментариями?

siriusdevs replied:
я понимаю, зачем в его сделали, но если он пустой, это подозрительно, как считаете?

siriusdevs commented on 01-11/models/geometric_shapes.py:
```diff
@@ -0,0 +1,167 @@
+"""Models of geometric shapes."""
+import math
+from .exeptions import UnrealTriangleError
```
exCeptions

siriusdevs commented on 01-11/models/exeptions.py:
```diff
@@ -0,0 +1,13 @@
+"""A few exceptions."""
```
в комментарии exceptions (правильно), а файл называется exeptions

siriusdevs commented on 01-11/models/exeptions.py:
```diff
@@ -0,0 +1,13 @@
+"""A few exceptions."""
+
+
+class UnrealTriangleError(Exception):
```
InvalidTriangleError? InvalidTriangleSides?

siriusdevs commented on 01-11/models/geometric_shapes.py:
```diff
@@ -0,0 +1,167 @@
+
+        Raises:
+            UnrealTriangleError: if triangle can't exist.
+            ValueError: if some sides not be float
```
if any of the sides is not float

siriusdevs commented on 01-11/models/geometric_shapes.py:
```diff
@@ -0,0 +1,167 @@
+        self.__side_a = float(side_a)
+        self.__side_b = float(side_b)
+        self.__side_c = float(side_c)
+        # проверка треугольника на существование
```
eto chto russkiy?

siriusdevs commented on 01-11/models/geometric_shapes.py:
```diff
@@ -0,0 +1,167 @@
+        Returns:
+            float - square of circle.
+        """
+        return round(math.pi * self.radius ** 2, 2)
```
остальное понравилось, молодец! :+1:

nazaroviktor2 replied:
🥰🥰🥰

siriusdevs commented on 01-11/test_geometric_shapes.py:
```diff
@@ -0,0 +1,96 @@
+        side_b: float - third side of triangle.
+    """
+    triangle = Triangle(side_a, side_b, side_c)
+    assert triangle.side_a == side_a
```
знаете, в чём тут проблема? эти 3 assert считаются за один тест. если первый будет false, остальные два не выполняются.  в данном случае это правильно, пожалуй, но на будущее обязательно имейте ввиду.

siriusdevs commented on 01-11/test_geometric_shapes.py:
```diff
@@ -0,0 +1,96 @@
+        radius: float - circle's radius.
+    """
+    circle = Circle(radius)
+    assert circle.radius == radius
```
assert Circle(radius).radius == radius ??

siriusdevs commented on setup.cfg:
```diff
@@ -1,5 +1,8 @@
 [flake8]
-ignore=Q000,WPS433,WPS407,C819,WPS306,I00,WPS300,E800,D106,WPS226,WPS360,C812,D401,WPS115,DAR201,N400,WPS317,E502,WPS202,WPS221,WPS318,WPS301,RST210
+ignore=Q000,WPS433,WPS504,WPS112,WPS614,WPS407,C819,WPS306,I00,WPS300,E800,D106,WPS226,WPS360,C812,D401,WPS115,DAR201,N400,WPS317,E502,WPS202,WPS221,WPS318,WPS301,RST210,S311,WPS337
```
не увлекайтесь, я слежу за вами :eyes: 
на этот раз ничего криминального, прощаю.

nazaroviktor2 replied:
eto chto russkiy?  :eyes:

siriusdevs commented:
#7 @nazaroviktor2 :timer_clock:

nazaroviktor2 committed:
Added tests for class cirlce.
Renamed file exeptions.py to exceptions.py.
Renamed exception UnrealTriangleError to InvalidTriangleSides.
Created new exception "InvalidCircleRadius" for error with class cirlce.
THE MOST IMPORTANT CHANGE: Russian text removed.
